jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -426,47 +500,20 @@ ecma_builtin_try_to_instantiate_property (ecma_object_t *object_p, /**< object *
 
   uint32_t index = (uint32_t) (curr_property_p - property_list_p);
 
-  JERRY_ASSERT (index < 64);
+  JERRY_ASSERT (index < 96);
```

Comment:
I mean we literally need to modify all of these instances, so if it would be a general constant, and it would be increased, we would not notice issues with the code.